pxr/imaging/hd/primTypeIndex.cpp

```diff
@@ -383,10 +384,15 @@ Hd_PrimTypeIndex<PrimType>::SyncPrims(HdChangeTracker  &tracker,
 
     _dirtyPrimDelegates.clear();
     HdSceneDelegate *prevDelegate = nullptr;
+
     for (size_t typeIdx = 0; typeIdx < numTypes; ++typeIdx) {
 
+        TfToken const& primType = _primTypeNames[typeIdx];
+
+        renderDelegate->WaitForEarlyInitTasks(primType);
```

Comment:
New render delegate API: optionally wait for parallel tasks associated with the particular prim type.